uiowa/uiowa
# Hotfix IP migration.

Related to #4812 

This should resolve `[error]  Error: Call to undefined method stdClass::save()` by just checking to see if we can load the block before processing. I'm unsure of the repercussions of this or what caused the error after it was fixed #4862 and then happening later in prod. Source data change? D9 code change?



# How to test

Sync latest source database on uiowa703 DD. Restore user-created backup locally for D9 IP site. Download it to the tmp directory within the application. 

```
ddev drush @international.local sqlc < tmp/backup-2022-04-05-20-08-international_uiowa_edu-205858668.sql
ddev blt drupal:update --site international.uiowa.edu
ddev drush rsync @international.prod:%files @international.local:%files -v --yes
ddev drush @international.local pmu stage_file_proxy
ddev drush @international.local en international_migrate --yes
ddev drush @international.local uli /admin/config/sitenow/sitenow-migrate
```

Configure the form for DD. 
```
ddev drush @international.local ms
ddev drush @international.local mim --all
```

Confirm both migrations complete without errors. Use the test criteria from https://github.com/uiowa/uiowa/pull/4862 to inspect the site. Double check no duplicate files created.
